test/test_utils/benchmark.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+
+def single_node_benchmark_executor(context, image_uri, framework, task, py_version):
+    """
```

Comment:
just curious, is there something useful we could return here? i.e. that can be used for logging/etc in the test function?